documentation/inventory_files.md

```diff
@@ -468,6 +468,145 @@ plugins:
         message: Enter your Vault password
 ```
 
+### Share inventory between projects
+
+Because projects are used to run a specific workflow on a list of targets, it's
+common to have multiple projects that each run a different workflow on the same
+list of targets. If you have multiple projects that all use the same inventory,
+it can be useful to use a shared inventory file so you don't need to maintain
+your inventory in each project individually.
+
+#### Using the command line
+
+The simplest way to specify a shared inventory is to save the inventory file in
+an accessible location such as the Bolt user configuration directory and then
+specify the inventory file from the command line.
+
+Save the shared inventory file to
+`~/.puppetlabs/etc/bolt/shared_inventory.yaml`:
+
+```yaml
+targets:
+  - server-1.example.org
+  - server-2.example.org
+  - server-3.example.org
+config:
+  ssh:
+    user: puppet
+    password: Bolt!
+    host-key-check: false
+```
+
+Specify the shared inventory file from the command line:
+
+_\*nix shell command_
+
+```shell
+bolt inventory show --targets all --inventoryfile ~/.puppetlabs/etc/bolt/shared_inventory.yaml
+```
+
+_PowerShell cmdlet_
+
+```powershell
+Get-BoltInventory -Targets 'all' -Inventoryfile 'C:\Users\Administrator\.puppetlabs\etc\bolt\shared_inventory.yaml'
+```
+
+#### Using the `yaml` plugin
+
+Instead of specifying the path to the shared inventory file for every command,
+you can load the shared inventory into each project's inventory dynamically.
+To do this, create an `inventory.yaml` file for each project and use the `yaml`
+plugin to load the shared inventory.
+
+Save the shared inventory file to
+`~/.puppetlabs/etc/bolt/shared_inventory.yaml`:
+
+```yaml
+targets:
+  - server-1.example.org
+  - server-2.example.org
+  - server-3.example.org
+config:
+  ssh:
+    user: puppet
+    password: Bolt!
+    host-key-check: false
+```
+
+Save an inventory file to each project that loads the shared inventory file:
+
+```yaml
+_plugin: yaml
+filepath: ~/.puppetlabs/bolt/shared_inventory.yaml
+```
+
+When the project loads the inventory, it injects the shared inventory file,
+allowing you to reference the same list of targets across multiple projects.
```

Comment:
Might be good to add a note about ordering here. If a target is specified multiple times the *first* appearance in the inventory file will be used, so this can be used to set "defaults" if it's the last block in the project inventoryfile, and then targets can be overridden with config, vars, and facts being merged.

Reply:
Added a new sub-section for specifying target defaults.